src/lib/core/tests/TestCATValues.cpp

```diff
@@ -0,0 +1,176 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    Copyright (c) 2013-2017 Nest Labs, Inc.
```

Comment:
```suggestion
```

Reply:
I wasn't sure if there was some claim about copying the NL boilerplate we have in all the tests.  But I think there is sufficient atribution in the NL headers.

Thanks, applied.